### ISSUE TITLE
Fix Missing Alt Text In Theme

### ISSUE THREAD

**Issue by adamdoe** [marketingincolor/mad-sunscape-website#1]
@eddt ,

Here is a list of missing alt text in the sunscape theme.

[Sunscape_MissingAltText_03072017.xlsx](https://github.com/marketingincolor/mad-sunscape-website/files/825245/Sunscape_MissingAltText_03072017.xlsx)

**Comment by adamdoe** [comment]
Edd I'm assigning this one to you since you're probably more familiar with the theme. However, it's something I can handle if you want me to look into it.

**Comment by adamdoe** [comment]
@eddt 

Low Priority.
Updated Missing alt text report:

P:\Clients\Madico\Website & Blogs\Sunscape_SEO
